packages/ceramic-cli/src/index.ts

```diff
@@ -1,163 +1,109 @@
-  .action(async (docId) => {
-    if (!validateDocId(docId)) {
-      console.error(`Invalid docId: ${docId}`)
-      return
-    }
-    const ceramic = new CeramicClient()
-    try {
-      const doc = await ceramic.loadDocument(docId)
-      console.log(JSON.stringify(DoctypeUtils.serializeState(doc.state), null, 2))
-    } catch (e) {
-      console.error(e)
-    }
-    ceramic.close()
-  })
+    .command('state <docId> [<anchor>]')
+    .description('Show the state of a document')
+    .action(async (docId) => {
+        await CeramicCliUtils.state(docId)
+    })
 
 program
-  .command('watch <docId>')
-  .description('Watch for updates in a document')
-  .action(async (docId) => {
-    if (!validateDocId(docId)) {
-      console.error(`Invalid docId: ${docId}`)
-      return
-    }
-    const ceramic = new CeramicClient()
-    try {
-      const doc = await ceramic.loadDocument(docId)
-      console.log(JSON.stringify(doc.content, null, 2))
-      doc.on('change', () => {
-        console.log('--- document changed ---')
-        console.log(JSON.stringify(doc.content, null, 2))
-      })
-    } catch (e) {
-      console.error(e)
-    }
-  })
+    .command('watch <docId>')
+    .description('Watch for updates in a document')
+    .action(async (docId) => {
+        await CeramicCliUtils.watch(docId)
+    })
 
 program
     .command('versions <docId>')
     .description('List document versions')
     .action(async (docId) => {
-        if (!validateDocId(docId)) {
-            console.error(`Invalid docId: ${docId}`)
-            return
-        }
-        const ceramic = new CeramicClient()
-        try {
-            const versions = await ceramic.listVersions(docId)
-            console.log(JSON.stringify(versions, null, 2))
-        } catch (e) {
-            console.error(e)
-        }
-        ceramic.close()
-    });
+        await CeramicCliUtils.versions(docId)
+    })
+
+const schemas = program.command('schema')
+schemas.description('Ceramic schemas')
+
+schemas
+    .command('create <new-content>')
+    .option('--only-genesis', 'Only create the genesis object. No anchor will be created')
+    .option('--owners <owners>', 'Specify a comma-separated list of the owners of the schema document. Defaults to' +
+        ' current user')
+    .option('--unique', 'Ensure schema document is unique regardless of content')
+    .description('Create a new schema')
+    .action(async (content, { onlyGenesis, owners, unique }) => {
+        await CeramicCliUtils.schemaCreateDoc(content, owners, onlyGenesis, unique)
+    })
+
+schemas
+    .command('change <docId> <new-content>')
+    .option('--owners <owners>', 'Change owner of this document (only 3ID)')
+    .description('Update the content of a schema')
+    .action(async (docId, content, { owners }) => {
+        await CeramicCliUtils.schemaChangeDoc(docId, content, owners)
+    })
+
+schemas
+    .command('validate <schemaDocId> <new-content>')
```

Comment:
I think these make sense. So basically it just creates a tile and also validates that it's adding an actual schema?

Reply:
yep, I think it's good for now, as well 👍